syntaxes/racket.tmLanguage.json

```diff
@@ -1316,7 +1318,7 @@
       "patterns": [
         {
           "name": "comment.sexp.racket",
-          "match": "(?<=^|[()\\[\\]{}\",'`;\\s])#;"
+          "match": "(?<=^|[()\\[\\]{}\\\",'`;\\s])#;"
```

Comment:
Is this change correct...? I think it's adding `\` to the character class. I can't quite work out if that makes sense.

Reply:
Thanks for catching this:

Here is the change I made to the yaml:
```diff
- match: "(?<=^|[()\[\]{}\",'`;\s])#;"
+ match: (?<=^|[()\[\]{}\",'`;\s])#;
```
The original problem I ran into was that js-yaml errored out here because \\[ and \\] and \\s are invalid escape sequences in a " quoted string. 
I removed the " quotes which causes js-yaml to properly escape the brackets and things, but forgot that I left the \" which caused \ to be added to the character class. We can either remove the \ from the yaml, or turn it back into a " quoted string and add extra \ before the brackets (i.e. \\\\[ \\\\]). I prefer to leave off the " quotes, which makes it something that you can just copy into a regex tester. I'll make that change.